Source/Scene/Model.js

```diff
@@ -1081,10 +1114,25 @@ define([
                 var shader = shaders[name];
 
                 // Shader references either uri (external or base64-encoded) or bufferView
-                if (defined(shader.extensions) && defined(shader.extensions.CESIUM_binary_glTF)) {
+                if (defined(shader.extras) && defined(shader.extras.source)) {
```

Comment:
This is OK for now.  I want to revisit an "in-memory glTF" where, for example, all the uris and replaced with typed arrays / GLSL strings.  This will be useful for creating glTF at runtime without crazy overhead and for the runtime data structure for an optimization pipeline.